Cargo.toml

```diff
@@ -14,23 +14,26 @@ documentation = "https://esp-rs.github.io/esp-idf-hal/"
 rust-version = "1.61"
 
 [features]
-default = ["std", "alloc", "esp-idf-sys"]
+default = ["std", "alloc", "esp-idf-sys", "async"]
```

Comment:
The feature in `esp-idf-svc` and `embedded-svc` that unlocks the async support is called... `nightly`. Maybe not the best name (comes from the fact that async support needs at least TAIT, and with the latest `embedded-hal-async` also `async-fn-in-trait` which are nightly features), but let's align and use `nightly` here as well.

Reply:
Will do